src/main/webapp/WEB-INF/tags/student/feedbackResults/response.tag

```diff
@@ -17,13 +18,8 @@
     <tr>
         <td>
             <ul class="list-group comment-list">                                                                           
-                <c:forEach items="${response.comments}" var="comment">                                                                            
-                    <li class="list-group-item list-group-item-warning" id="responseCommentRow-${comment.frcId}">
-                        <div id="commentBar-${comment.frcId}">
-                            <span class="text-muted">From: ${comment.giverEmail} [${comment.createdAt}] ${comment.editedAtText}</span>
-                        </div>
-                        <div id="plainCommentText-${comment.frcId}" style="margin-left: 15px;">${comment.commentText}</div>
-                    </li>
+                <c:forEach items="${response.comments}" var="comment">
+                    <shared:feedbackResponseComment frc="${comment}" />
```

Comment:
This seems to add a lot of elements to the unodered list, was response.tag previously unused?